themes/mlibrary_new/items/show.php

```diff
@@ -1,67 +1,106 @@
 <?php
-fire_plugin_hook('public_items_show', array('view' => $this, 'item' => $item));
-
-// The page isn't injected by the controller
-$page = get_db()->getTable('ExhibitPage')->find($_GET['page']);
+ /**
+  * Copyright (c) 2016, Regents of the University of Michigan.
+  * All rights reserved. See LICENSE.txt for details.
+  */
+  $item_title = strip_formatting(metadata('item', array('Dublin Core', 'Title')));
+  if (empty($item_title)) { $item_title = __('[Untitled]'); }
+  echo head(
+    array(
+      'title'     => $item_title,
+      'bodyid'    => 'items',
+      'bodyclass' => 'show item'
+    )
+  );
+ $exhibit_page = get_record_by_id('exhibit_page',html_escape($_GET['page']));
```

Comment:
Why are you running `html_escape` on the argument to `get_record_by_id` ?

Reply:
We should definitely run `html_escape()` when printing something out on the screen, yes.

But `get_record_by_id()` is doing a database lookup, right?  The database ids shouldn't be html, they are probably going to be integers though, so `intval()` might be appropriate.

Reply:
ok